functions/go/remove-local-config-resources/README.md

```diff
@@ -0,0 +1,89 @@
+# remove-local-config-resources
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Removes resources with the annotation `config.kubernetes.io/local-config: true` from the resource list.
+
+<!--mdtogo-->
+
+This function provides a quick way for users to prune resources that are only intended to work client-side.
+For example, it can be useful for Config Sync to avoid picking up client-side resources for validation and hydration.
+
+<!--mdtogo:Long-->
+
+## Usage
+
+`remove-local-config-resources` function can be executed imperatively as follows:
+
+```shell
+$ kpt fn eval -i gcr.io/kpt-fn/remove-local-config-resources:unstable
+```
+
+The function will execute as follows:
```

Comment:
I'd suggest move these steps to the top of the Usage section.
So the usage section has the following order:
1) steps how it works.
2) Imperative
3) declarative

Reply:
Done.